[2019-05-01 15:18:09,646] {__init__.py:1139} INFO - Dependencies all met for <TaskInstance: Salted_Graph_Practice1.7.Generate_Transform.hash 2019-05-01T19:18:00.962939+00:00 [queued]>
[2019-05-01 15:18:09,649] {__init__.py:1139} INFO - Dependencies all met for <TaskInstance: Salted_Graph_Practice1.7.Generate_Transform.hash 2019-05-01T19:18:00.962939+00:00 [queued]>
[2019-05-01 15:18:09,649] {__init__.py:1353} INFO - 
--------------------------------------------------------------------------------
[2019-05-01 15:18:09,649] {__init__.py:1354} INFO - Starting attempt 1 of 1
[2019-05-01 15:18:09,649] {__init__.py:1355} INFO - 
--------------------------------------------------------------------------------
[2019-05-01 15:18:09,658] {__init__.py:1374} INFO - Executing <Task(PythonOperator): 7.Generate_Transform.hash> on 2019-05-01T19:18:00.962939+00:00
[2019-05-01 15:18:09,658] {base_task_runner.py:119} INFO - Running: ['airflow', 'run', 'Salted_Graph_Practice1', '7.Generate_Transform.hash', '2019-05-01T19:18:00.962939+00:00', '--job_id', '344', '--raw', '-sd', 'DAGS_FOLDER/salted_graph.py', '--cfg_path', '/tmp/tmp3jarrj_x']
[2019-05-01 15:18:09,934] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash /home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/configuration.py:214: FutureWarning: The task_runner setting in [core] has the old default value of 'BashTaskRunner'. This value has been changed to 'StandardTaskRunner' in the running config, but please update your config before Apache Airflow 2.0.
[2019-05-01 15:18:09,934] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash   FutureWarning,
[2019-05-01 15:18:10,080] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash [2019-05-01 15:18:10,079] {__init__.py:51} INFO - Using executor SequentialExecutor
[2019-05-01 15:18:10,218] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash [2019-05-01 15:18:10,218] {__init__.py:305} INFO - Filling up the DagBag from /home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/dags/salted_graph.py
[2019-05-01 15:18:10,359] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash [2019-05-01 15:18:10,359] {cli.py:517} INFO - Running <TaskInstance: Salted_Graph_Practice1.7.Generate_Transform.hash 2019-05-01T19:18:00.962939+00:00 [running]> on host localhost.localdomain
[2019-05-01 15:18:10,364] {python_operator.py:104} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_ID=Salted_Graph_Practice1
AIRFLOW_CTX_TASK_ID=7.Generate_Transform.hash
AIRFLOW_CTX_EXECUTION_DATE=2019-05-01T19:18:00.962939+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2019-05-01T19:18:00.962939+00:00
[2019-05-01 15:18:10,366] {__init__.py:1580} ERROR - [Errno 2] File b'/home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/copied_data/yelp_subset_1.034bc9d2.csv' does not exist: b'/home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/copied_data/yelp_subset_1.034bc9d2.csv'
Traceback (most recent call last):
  File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/models/__init__.py", line 1441, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/operators/python_operator.py", line 112, in execute
    return_value = self.execute_callable()
  File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/operators/python_operator.py", line 117, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/dags/salted_graph.py", line 245, in transform_data
    df = pd.read_csv(copied_file_path)
  File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/pandas/io/parsers.py", line 702, in parser_f
    return _read(filepath_or_buffer, kwds)
  File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/pandas/io/parsers.py", line 429, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/pandas/io/parsers.py", line 895, in __init__
    self._make_engine(self.engine)
  File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/pandas/io/parsers.py", line 1122, in _make_engine
    self._engine = CParserWrapper(self.f, **self.options)
  File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/pandas/io/parsers.py", line 1853, in __init__
    self._reader = parsers.TextReader(src, **kwds)
  File "pandas/_libs/parsers.pyx", line 387, in pandas._libs.parsers.TextReader.__cinit__
  File "pandas/_libs/parsers.pyx", line 705, in pandas._libs.parsers.TextReader._setup_parser_source
FileNotFoundError: [Errno 2] File b'/home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/copied_data/yelp_subset_1.034bc9d2.csv' does not exist: b'/home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/copied_data/yelp_subset_1.034bc9d2.csv'
[2019-05-01 15:18:10,367] {__init__.py:1611} INFO - Marking task as FAILED.
[2019-05-01 15:18:10,381] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash Traceback (most recent call last):
[2019-05-01 15:18:10,381] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash   File "/home/mjcarleb/anaconda3/envs/airflow4/bin/airflow", line 32, in <module>
[2019-05-01 15:18:10,381] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash     args.func(args)
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/utils/cli.py", line 74, in wrapper
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash     return f(*args, **kwargs)
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/bin/cli.py", line 523, in run
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash     _run(args, dag, ti)
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/bin/cli.py", line 442, in _run
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash     pool=args.pool,
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/utils/db.py", line 73, in wrapper
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash     return func(*args, **kwargs)
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/models/__init__.py", line 1441, in _run_raw_task
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash     result = task_copy.execute(context=context)
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/operators/python_operator.py", line 112, in execute
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash     return_value = self.execute_callable()
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/operators/python_operator.py", line 117, in execute_callable
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash     return self.python_callable(*self.op_args, **self.op_kwargs)
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash   File "/home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/dags/salted_graph.py", line 245, in transform_data
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash     df = pd.read_csv(copied_file_path)
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/pandas/io/parsers.py", line 702, in parser_f
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash     return _read(filepath_or_buffer, kwds)
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/pandas/io/parsers.py", line 429, in _read
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash     parser = TextFileReader(filepath_or_buffer, **kwds)
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/pandas/io/parsers.py", line 895, in __init__
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash     self._make_engine(self.engine)
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/pandas/io/parsers.py", line 1122, in _make_engine
[2019-05-01 15:18:10,382] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash     self._engine = CParserWrapper(self.f, **self.options)
[2019-05-01 15:18:10,383] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/pandas/io/parsers.py", line 1853, in __init__
[2019-05-01 15:18:10,383] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash     self._reader = parsers.TextReader(src, **kwds)
[2019-05-01 15:18:10,383] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash   File "pandas/_libs/parsers.pyx", line 387, in pandas._libs.parsers.TextReader.__cinit__
[2019-05-01 15:18:10,383] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash   File "pandas/_libs/parsers.pyx", line 705, in pandas._libs.parsers.TextReader._setup_parser_source
[2019-05-01 15:18:10,383] {base_task_runner.py:101} INFO - Job 344: Subtask 7.Generate_Transform.hash FileNotFoundError: [Errno 2] File b'/home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/copied_data/yelp_subset_1.034bc9d2.csv' does not exist: b'/home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/copied_data/yelp_subset_1.034bc9d2.csv'
[2019-05-01 15:18:14,650] {logging_mixin.py:95} INFO - [2019-05-01 15:18:14,650] {jobs.py:2562} INFO - Task exited with return code 1
